### DISCUSSION TITLE
[ Common ] Performance with Direct3D11 significantly poorer than OpenGL

### DISCUSSION THREAD

**Discussion by DLPB2** [julianxhokaxhiu/FFNx#193] [General]
Frame rate tests show this - although it could also be my graphic card and so on.  Worth testing.

**Comment by julianxhokaxhiu** [comment]
Hi,

Sorry but this has nothing to do with FFNx. The renderer is on top of BGFX so feel free to open an issue on their repo :)

Cheers!
